tuna/miopen/celery_tuning/tuning.py

```diff
@@ -290,102 +291,75 @@ def tune(library, job_batch_size=1000):
       subp.kill()
     return False
 
-  res_set = ResultSet([])
+  #res_set = ResultSet([])
   start = time.time()
   worker_type = get_worker_type(library.args)
-  drain_process = results_gather_start(worker_type)
-
+  
+  db_name = os.environ['TUNA_DB_NAME']
+  prefix = f"d_{db_name}_sess_{library.args.session_id}"
+  app.conf.get('result_backend_transport_options', {}).update({"global_keyprefix": prefix})
   with DbSession() as session:
-    while True:
-      try:
-        job_list = []
-        #get all the jobs from mySQL
-        job_list = library.get_jobs(session, library.fetch_state,
-                                    library.set_state, library.args.session_id,
-                                    job_batch_size)
-
-        for i in range(0, len(job_list), job_batch_size):
-          batch_jobs = job_list[i:min(i + job_batch_size, len(job_list))]
-          entries = library.compose_work_objs_fin(session, batch_jobs,
-                                                  library.dbt)
-          serialized_jobs = serialize_chunk(entries)
-          #build context for each celery task
-          for job, config in serialized_jobs:
-            context = {
-                'job': job,
-                'config': config,
-                'worker_type': worker_type,
-                'arch': library.dbt.session.arch,
-                'num_cu': library.dbt.session.num_cu,
-                'kwargs': kwargs,
-                'fdb_attr': fdb_attr
-            }
-
-            #calling celery task, enqueuing to celery queue
-            res_set.add(
-                celery_enqueue.apply_async((context,),
-                                           queue=q_name,
-                                           reply_to=q_name))
-
-        if not job_list:
-          if not res_set:
-            results_gather_terminate(res_set, drain_process)
-            return False
-          LOGGER.info('All tasks added to queue')
-          j_end = time.time()
-          break
-      except KeyboardInterrupt:
-        LOGGER.error('Keyboard interrupt caught, draining results queue')
-        session.rollback()
-        results_gather_terminate(res_set, drain_process)
-        purge_queue([q_name])
-        library.cancel_consumer(q_name)
-
-  results_gather_terminate(res_set, drain_process)
-  library.cancel_consumer(q_name)
-  end = time.time()
-  LOGGER.info("Took {:0>8} to tune".format(str(timedelta(seconds=end - start))))  #pylint: disable=consider-using-f-string
-  LOGGER.info("{:0>8} of which was spent enqueuing jobs".format(  #pylint: disable=consider-using-f-string
-      str(timedelta(seconds=j_end - start))))
+    job_list = library.get_jobs(session, library.fetch_state,
```

Comment:
make sure there is logging in this function to indicate progress.

Reply:
Perhaps the query can be simplified so that its a `SELECT count(*)` instead of `select *`.

Reply:
i was just piggybacking off a function we already have